packages/@okta/vuepress-site/docs/concepts/policies/index.md

```diff
@@ -21,29 +21,29 @@ There are many possibilities for policy use:
 
 Okta supports the following policy types:
 
-* [Okta Sign On Policy](/docs/reference/api/policy/#okta-sign-on-policy) &mdash; Controls who can sign in and how a user is allowed to sign in to Okta, including whether they are challenged for MFA and how long they are allowed to remain signed in before re-authenticating.
+* [Global Session Policy](/docs/reference/api/policy/#global-session-policy): Controls who can sign in and how a user is allowed to sign in to Okta, including whether they are challenged for MFA and how long they are allowed to remain signed in before re-authenticating.
```

Comment:
Is the heading "Okta sign-on policy" going to change to "Global Session policy" in the `/docs/reference/api/policy` page?